scrubber_test.go

```diff
@@ -114,3 +150,86 @@ func TestScrubbing(t *testing.T) {
 		compareImages(t, test.originalFile, test.expectedFile)
 	}
 }
+
+func updateCorpus(logger io.Writer) error {
+	currentETag, err := ioutil.ReadFile(corpusETagFile)
+	if err != nil && !os.IsNotExist(err) {
+		return fmt.Errorf("failed to read current ETag file: %w", err)
+	}
+	headResp, err := http.Head(corpusURL)
+	if err != nil {
+		return fmt.Errorf("HEAD failed: %w", err)
+	}
+	defer headResp.Body.Close()
+	if headResp.StatusCode != http.StatusOK {
+		return fmt.Errorf("bad response to HEAD request: %s", headResp.Status)
+	}
+	etag := headResp.Header.Get("ETag")
+	if etag == "" {
+		return errors.New("no ETAG in HEAD response")
+	}
+	if etag == strings.TrimSpace(string(currentETag)) {
```

Comment:
This is a bit of a hack.  In CircleCI, the ETag file will contain a trailing newline so we trim the file of space characters just in case.